packages/lit-dev-content/site/docs/templates/directives.md

```diff
@@ -11,33 +11,228 @@ versionLinks:
+  </tr>
+
+  <tr class="subheading"><td colspan="2">
+
+  Loops and Conditionals
+
+  </td></tr>
+
+  <tr>
+  <td>
+
+  [`when`](#when)
+
+  </td>
+  <td>Renders one of two templates based on a condition.</td>
+  </tr>
+
+  <tr>
+  <td>
+
+  [`choose`](#choose)
+
+  </td>
+  <td>Renders one of many templates based on a key value.</td>
+  </tr>
+
+  <tr>
+  <td>
+
+  [`map`](#map)
+
+  </td>
+  <td>Transforms an iterable with a function.</td>
+  </tr>
+
+  <tr>
+  <td>
+
+  [`repeat`](#repeat)
+
+  </td>
+  <td>Renders values from an iterable into the DOM, with optional keying to enable data diffing and DOM stability.</td>
+  </tr>
+
+  <tr>
+  <td>
+
+  [`join`](#join)
+
+  </td>
+  <td>Interleave values from an iterable with a joiner value.</td>
+  </tr>
+
+  <tr>
+  <td>
+
+  [`range`](#range)
+
+  </td>
+  <td>Creates an iterable of numbers in a sequence, useful for iterating a specific number of times.</td>
+  </tr>
+
+  <tr>
+  <td>
+
+  [`ifDefined`](#ifdefined)
+
+  </td>
+  <td>Sets an attribute if the value is defined and removes the attribute if undefined.</td>
+  </tr>
+
+  <tr class="subheading"><td colspan="2">
+
+  Caching and change detection
+
+  </td></tr>
+
+  <tr>
+  <td>
+
+  [`cache`](#cache)
+
+  </td>
+  <td>Caches rendered DOM when changing templates rather than discarding the DOM.</td>
+  </tr>
+
+  <tr>
+  <td>
+
+  [`guard`](#guard)
+
+  </td>
+  <td>Only re-evaluates the template when one of its dependencies changes.</td>
+  </tr>
+
+  <tr>
+  <td>
+
+  [`live`](#live)
+
+  </td>
+  <td>Sets an attribute or property if it differs from the live DOM value rather than the last-rendered value.</td>
+  </tr>
+
+  <tr class="subheading"><td colspan="2">
+
+  Referencing rendered DOM
+
+  </td></tr>
+
+  <tr>
+  <td>
+
+  [`ref`](#ref)
+
+  </td>
+  <td>Gets a reference to an element rendered in the template.</td>
+  </tr>
+
+  <tr class="subheading"><td colspan="2">
+
+  Rendering special values
+
+  </td></tr>
+
+  <tr>
+  <td>
+
+  [`templateContent`](#templatecontent)
+
+  </td>
+  <td>
+
+  Renders the content of a `<template>` element.
+
+  </td>
+  </tr>
+
+  <tr>
+  <td>
+
+  [`unsafeHTML`](#unsafehtml)
+
+  </td>
+  <td>Renders a string as HTML rather than text.</td>
+  </tr>
+
+  <tr>
+  <td>
+
+  [`unsafeSVG`](#unsafesvg)
+
+  </td>
+  <td>Renders a string as SVG rather than text.</td>
+  </tr>
+
+  <tr class="subheading"><td colspan="2">
+
+  Asynchronous rendering
+
+  </td></tr>
+
+  <tr>
+  <td>
+
+  [`until`](#until)
+
+  </td>
+  <td>Renders placeholder content until one or more promises resolve.</td>
+  </tr>
+
+  <tr>
+  <td>
+
+  [`asyncAppend`](#asyncappend)
+
+  </td>
+  <td>Appends values from an `AsyncIterable` into the DOM as they are yielded.</td>
+  </tr>
+
+  <tr>
+  <td>
+
+  [`asyncReplace`](#asyncreplace)
+
+  </td>
+  <td>Renders the latest value from an `AsyncIterable` into the DOM as it is yielded.</td>
+  </tr>
+</table>
```

Comment:
@graynorton note on the blog post made me think mayve we should emphasize the pay-as-you-go nature of directives here, too?

```suggestion
</table>

<div class="alert alert-info">

**Only bundle what you use.** These are called "built-in" directives because they're part of the Lit package. But each directive is a separate module, so your app only bundles the directives you import.

</div>

```